core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh

```diff
@@ -60,6 +60,17 @@ create_directory() {
 	fi
 }
 
+# params: <seconds>
+display_seconds() {
+	local seconds=$1
+	local h=$((seconds/3600))
+	local m=$((seconds%3600/60))
+	local s=$((seconds%60))
+	(( h > 0 )) && printf '%dh ' $h
+	(( m > 0 )) && printf '%dm ' $m
+	               printf '%ds' $s
+}
+
```

Comment:
This is an overkill. I'd suggest single `date` invocation: https://stackoverflow.com/a/13422982